libraries/Microsoft.Bot.Builder.Ai.LUIS/LuisRecognizer.cs

```diff
@@ -121,13 +100,24 @@ private static JObject GetEntitiesAndMetadata(IList<EntityRecommendation> entiti
 
                 if (verbose)
                 {
-                    AddProperty((JObject) entitiesAndMetadata[MetadataKey], GetNormalizedEntityType(entity), GetEntityMetadata(entity));
+                    AddProperty((JObject)entitiesAndMetadata[MetadataKey], GetNormalizedEntityType(entity), GetEntityMetadata(entity));
                 }
             }
 
             return entitiesAndMetadata;
         }
 
+        private static JToken Number(dynamic value)
+        {
+            if (value == null)
+            {
+                return null;
+            }
+            return long.TryParse((string)value, out var longVal) ?
```

Comment:
I thought you wanted to get rid of this and always return double? I'm ok with keeping it

Reply:
I map to double in the strong conversion--fine to leave it either here in JSON.